Topics/Tech_Stacks/Odata.md

```diff
@@ -0,0 +1,121 @@
+# Learning OData for RESTful API Development
+
+## Table of contents
+
+### [Introduction](#introduction-1)
+
+### [What is OData](#what-is-odata-1)
+
+### [Getting Started with OData](#getting-started-with-odata-1)
+
+### [OData Basics](#odata-basics-1)
+
+### [How to Use OData for RESTful APIs](#odata-basics-1)
```

Comment:
The “How to Use OData for RESTful APIs” table of contents title isn’t linking to the correct section

Reply:
Oops, I will fix that. Thanks.